storage_service/templates/snippets/locations_table.html

```diff
@@ -51,7 +53,9 @@
         <td>{{ loc.used }}B / {{ loc.quota|default:_("unlimited") }}</td>
         <td>{{ loc.enabled|yesno:_("Enabled,Disabled") }}</td>
         <td>{{ loc.default|yesno:_("Yes,No") }}</td>
-        <td><a href="{% url 'locations:location_edit' loc.space.uuid loc.uuid %}">{% trans "Edit" %}</a> | <a href="{% url 'locations:location_switch_enabled' loc.uuid %}?next={{ request.path }}">{{ loc.enabled|yesno:_("Disable,Enable") }}</a> | <a href="{% url 'locations:location_delete' loc.uuid %}?next={{ request.path }}">{% trans "Delete" %}</a></td>
+        {% if perms.locations.change_location %}
```

Comment:
Should the `Delete` link contained here be protected with the `locations.delete_location` permission?